tests/core/snap-set-core-config/task.yaml

```
@@ -162,3 +164,20 @@ execute: |
     hostname | MATCH foo
     snap set system system.hostname=F00
     hostnamectl status | MATCH F00
+
+    echo "configuring ssh access"
+    ssh-keygen -t rsa -N "" -f "$HOME"/.ssh/id_rsa
+    cat "$HOME"/.ssh/id_rsa.pub >> /home/ubuntu/.ssh/authorized_keys
+    # For some reason this directory has the wrong user on UC18
+    chown ubuntu:ubuntu /home/ubuntu
+
+    echo "ssh is enabled and works by default"
+    ssh -o "StrictHostKeyChecking no" ubuntu@localhost pwd
+    echo "ssh can be disabled"
+    snap set system service.ssh.disable=true
+    not ssh -o "StrictHostKeyChecking no" ubuntu@localhost pwd
```

you can use this here and in the other 2 places where ssh is called:
```suggestion
    not ssh ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no ubuntu@localhost pwd
```

---

Nice suggestion, done now